src/tir/ir/tir_visitor_with_path.cc

```diff
@@ -195,6 +204,10 @@ void TIRVisitorWithPath::VisitStmt_(const AttrStmtNode* op, ObjectPath path) {
     Visit(expr.value(), path->Attr("node"));
   }
   Visit(op->body, path->Attr("body"));
+
+  while (context.size()) {
```

Comment:
Thanks for the change. It's probably worth also putting in a line that this is to ensure that the defs expire, otherwise it seems like spooky action at a distance.

Reply:
Yeah, I tried to follow the existing `FooContext` naming structure (e.g. `arith::ConstraintContext`).